graphdatascience/tests/integration/test_graph_ops.py

```diff
@@ -101,6 +101,21 @@ def test_sample_rwr(runner: QueryRunner, gds: GraphDataScience) -> None:
     runner.run_query(f"CALL gds.graph.drop('{rwr_G.name()}')")
 
 
+@pytest.mark.compatible_with(min_inclusive=ServerVersion(2, 2, 0))
```

Comment:
This should be `@pytest.mark.compatible_with(min_inclusive=ServerVersion(2, 4, 0))` since GDS 2.4.0 is the version this algorithm is being added